Dont openbsd dev/null, doesnt exist in the default chroot

--- git/setup.c.orig	Wed Mar  1 21:01:06 2017
+++ git/setup.c	Wed Mar  1 21:01:56 2017
@@ -1024,6 +1024,7 @@ const char *resolve_gitdir(const char *suspect)
 /* if any standard file descriptor is missing open it to /dev/null */
 void sanitize_stdfds(void)
 {
+#if 0
 	int fd = open("/dev/null", O_RDWR, 0);
 	while (fd != -1 && fd < 2)
 		fd = dup(fd);
@@ -1031,6 +1032,7 @@ void sanitize_stdfds(void)
 		die_errno("open /dev/null or dup failed");
 	if (fd > 2)
 		close(fd);
+#endif
 }
 
 int daemonize(void)
